src/Domain/Entities/UserCredential.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Domain.Common;
+
+namespace Domain.Entities
+{
+    class UserCredential : BaseEntity
+    {
+        public string Email { get; set; }
+        public string Password { get; set; }
+        public string Role { get; set; }
+    }
```

Comment:
Didn't we include `public DateTime LastLogin { get; set; }` @deba1 ?

Reply:
We should add `LastLogin`. It was not in previous code.
Yes. I didn't add any virtual properties, as not all Entities were ready.